pkg/localstore/mode_put.go

```diff
@@ -213,7 +213,7 @@ func (db *DB) putRequest(batch *leveldb.Batch, binIDs map[uint8]uint64, item she
 		// if a chunk is found with the same postage stamp index,
 		// replace it with the new one only if timestamp is later
 		if !later(previous, item) {
-			return false, 0, err
+			return false, 0, nil
```

Comment:
I wonder if we need to have an explicit error here. @zelig?

Reply:
no error needed